.github/workflows/hydroshare-pull-requests.yml

```diff
@@ -36,21 +36,3 @@ jobs:
     - name: Run HydroShare Tests
       run: |
         docker exec hydroshare ./run-tests-jenkins
-    
-    - name: Publish Test Results
-      uses: EnricoMi/publish-unit-test-result-action/linux@v2
-      if: always()
-      with:
-        files: |
-          nosetests.xml
-          pytests.xml
```

Comment:
I couldn't get this GH action to work with the updated (non "nose") test runner :(
I attempted to install and use https://github.com/xmlrunner/unittest-xml-reporting to create the xml in the format required for publishing but it wasn't working out, so I just removed the publishing action